web/build.gradle

```diff
@@ -0,0 +1,7 @@
+dependencies {
+    compile group: 'org.springframework', name: 'spring-webmvc', version: '5.3.3'
+    compile group: 'mysql', name: 'mysql-connector-java', version: '8.0.22'

```

Comment:
DAO dependency in WEB.

Reply:
sorry again. feeling stupid. thanks for noticing this